packages/ui/src/styles/_root.sass

```diff
@@ -27,15 +27,18 @@ $cui-buildState-alreadyOutputRoot: false !default
 		margin: 0
 		padding: 0
 
+	html
+		display: flex
+		flex-direction: column
+		min-width: fit-content
+
 	body
+		flex: 1 1 100%
```

Comment:
Proč je tohle potřeba? Co by se stalo, kdyby se to nenastavilo?

Reply:
Visual regression since default `flex-base: 0%`, which means the flex starts at base 0% of parent towards whatever content requires with respect to `flex-grow` and `flex-base. Setting `flex-base` to 100% starts at 100% width of parent. It's like giving flex layout engine hint what layout you expect.

<img width="1680" alt="Screenshot 2022-02-07 at 14 51 36" src="https://user-images.githubusercontent.com/387611/152800760-9301b3d0-9cb9-416d-8ff9-45f5ce208993.png">